lib/nx_inflate.c

```diff
@@ -1280,8 +1301,11 @@ static int nx_inflate_(nx_streamp s, int flush)
 
 	/* fault in pages */
 	nx_touch_pages_dde(ddl_in, source_sz, nx_config.page_sz, 0);
-	nx_touch_pages_dde(ddl_out, target_sz, nx_config.page_sz, 1);
```

Comment:
This line doesn't need changes, but I'm ok with it.

Reply:
Good point. A previous change didn't work well and I forgot to revert the indentation changes.